nimble/host/src/ble_l2cap_coc.c

```diff
@@ -186,10 +189,12 @@ ble_l2cap_coc_rx_fn(struct ble_l2cap_chan *chan)
     rx = &chan->coc_rx;
     BLE_HS_DBG_ASSERT(rx != NULL);
 
+    rx_sdu = rx->sdus[current_sdu];
```

Comment:
are we sure tha rx_sdu will be non null?

Reply:
btw - please add current_sdu to debug logs somewhere below

Reply:
fixed